challenge_serving_drinks/src/challenge_serving_drinks/drive_in.py

```diff
@@ -0,0 +1,88 @@
+# ROS
+import smach
+
+# Robot smach states
+import robot_smach_states as states
+from robocup_knowledge import knowledge_loader
+
+# Serving drinks
+from .sd_states import AskAvailability
+
+# Knowledge
+CHALLENGE_KNOWLEDGE = knowledge_loader.load_knowledge("challenge_serving_drinks")
+
+
+class DriveIn(smach.StateMachine):
+    """ Serves on drink to an operator
+
+    """
+    def __init__(self, robot, bar_designator, room_designator, objects_list_des, unav_drink_des):
+        """ Initialization method
+
+        :param robot: robot api object
+        """
+        smach.StateMachine.__init__(self, outcomes=["succeeded", "failed", "aborted"])
```

Comment:
For completeness, you could check the types and write-ability of each designator. Will help you catch bugs

Reply:
@SamAlexandrov fix this or create issues for it.

Reply:
Added to the checklist in #838